.github/workflows/linux-workflow.yml

```diff
@@ -95,22 +97,62 @@ jobs:
         with:
           path: .ccache
           key: ${{ matrix.os }}-${{ matrix.platform }}-${{ matrix.compiler }}${{ matrix.detail }}-ccache-${{ steps.ccache_cache_timestamp.outputs.timestamp }}
-          restore-keys: |
-            ${{ matrix.os }}-${{ matrix.platform }}-${{ matrix.compiler }}${{ matrix.detail }}-ccache-
+          restore-keys: ${{ matrix.os }}-${{ matrix.platform }}-${{ matrix.compiler }}${{ matrix.detail }}-ccache-
```

Comment:
For some reason matrix.detail and matrix.cache have a space there, and appimage has no dash

Reply:
https://github.com/PCSX2/pcsx2/pull/4529 is going to remove the dashes, so i had started to, but since it is a separate PR, i left it alone. I removed the dash from `detail: appimage` since it has to match the `if...` statements for building the appimage artifacts and uploading them.

https://github.com/PCSX2/pcsx2/blob/db08ddb10410a73a846da62ac010ef827b8a75ee/.github/workflows/linux-workflow.yml#L151
https://github.com/PCSX2/pcsx2/blob/db08ddb10410a73a846da62ac010ef827b8a75ee/.github/workflows/linux-workflow.yml#L158

Reply:
Seems fine to me if another PR is taking care of it, all good then!